easybuild/easyconfigs/s/snpEff/snpEff-4.0.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'Tarball'
+
+name = 'snpEff'
+version = '4.0'
+
+homepage = 'http://snpeff.sourceforge.net'
+description = """Genetic variant annotation and effect prediction toolbox."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['{0}_v{1}_{2}_core.zip'.format(name, *version.split('.'))]
```

Comment:
we typically use `%s` style string formatting, so maybe stick with that?
also, use the provided templates

```python
sources = ['%(name)s_v%(version_major)s_%(version_minor)s_core.zip']
```